src/neoxp/Extensions/FileSystemExtensions.cs

```diff
@@ -65,10 +65,7 @@ public static string GetNodePath(this IFileSystem fileSystem, ExpressConsensusNo
 
             var account = node.Wallet.Accounts.Single(a => a.IsDefault);
 
-            var rootPath = fileSystem.Path.Combine(
-                Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.DoNotVerify),
-                "Neo-Express",
-                "blockchain-nodes");
+            var rootPath = "blockchain-nodes";
```

Comment:
```suggestion
            var homeDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile, Environment.SpecialFolderOption.DoNotVerify);
            var expressBlockchainDir = fileSystem.Path.Combine(homeDir, ".neo-express", "blockchain-nodes", account.ScriptHash);

            var appData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.DoNotVerify);
            var expressOldBlockchainDir = fileSystem.Path.Combine(appData, "Neo-Express", "blockchain-nodes", account.ScriptHash);

            if (fileSystem.Directory.Exists(expressOldBlockchainDir))
                return expressOldBlockchainDir;

            return expressBlockchainDir;
```

Updated to work with #324 and be backwards compatible